lib/tasks/tiny_mce_asset.rake

```diff
@@ -40,13 +53,42 @@ namespace :tiny_mce_asset do
       step.save
     end
     puts 'Completed processing steps...'
+
+    puts '----------- TASK REPORT -----------------'
+    puts "All images: #{all_images}"
+    puts "Recreated images: #{replaced_images}"
+    puts "Failed attempts: #{failed_attempts}"
+    puts "TinyMceAsset ids of failed attempts: #{failed_attempts_ids}"
+    puts '-----------------------------------------'
+  end
+
+  desc 'Generate new tiny_mce_assets and replace old assets in RTE ' \
+       'for results. Assign the last printed id if the script crashes or ' \
+       'id + 1 if there is a problematic asset'
+  task :regenerate_results_images, [:last_id] => :environment do |_, args|
+    regex = /\[~tiny_mce_id:([0-9a-zA-Z]+)\]/
```

Comment:
If it's the same regex as for steps, you can maybe just define it as constant on the `namespace` level.